snap-slack-dark-mode.sh

```diff
@@ -58,6 +55,9 @@ if [[ -f custom-dark-theme.css ]]; then
 fi
 
 if [[ "$UPDATE_ONLY" == "false" ]]; then
+    # ensure we don't have the mount already
+    unmount_slack
```

Comment:
If you're wondering about this change, its basically so it only kills slack completely if we're trying to pack the asar.